app/src/ui/diff/side-by-side-diff.tsx

```diff
@@ -382,20 +379,6 @@ export class SideBySideDiff extends React.Component<
     const lineFilters = getLineFilters(currentDiff.hunks)
     const tabSize = 4
 
-    const contents =
-      currentContents ?? (await getFileContents(repository, file, lineFilters))
-
-    if (
-      !highlightParametersEqual(
-        this.props,
-        propsSnapshot,
-        this.state,
-        stateSnapshot
-      )
-    ) {
-      return
-    }
```

Comment:
Could you elaborate for a bit on why this isn't needed anymore? It seems to me it could still be useful to bail here and not kick off another set of highlighter workers?

Reply:
Unless I'm missing something, that was useful only because we loaded the file contents here, right?

I mean, that `highlightParametersEqual` was supposed to bail if the diff changed while the app was loading the file contents. With this PR, the file contents are not loaded here anymore, but in `SeamlessDiffSwitcher`, and that's where we're bailing: https://github.com/desktop/desktop/pull/12903/files#diff-c771a816631b2d7e1340655981ed97791b724216730e5353a624356ee692f5b0R245-R247

Does that make sense? 😳

Reply:
> Does that make sense? 😳

It sure does, thanks!